plugins/git4idea/src/git4idea/branch/GitRenameBranchOperation.java

```diff
@@ -80,6 +83,7 @@ protected void rollback() {
       myNotifier.notifySuccess(BRANCH_RENAME_ROLLBACK_SUCCESS,
                                GitBundle.message("git.rename.branch.rollback.successful"),
                                GitBundle.message("git.rename.branch.renamed.back.to", myCurrentName));
+      notifyBranchNameRollback(myNewName, myCurrentName);
```

Comment:
It's better to choose and either call listener before notification or after it.
Should not change anything, but consistence is generally good.

Reply:
Fixed.